frontend/src/components/regions/CardDetails/index.js

```diff
@@ -108,12 +108,10 @@ function CardDetailsUnconnected({
   const isAssignee =
     ((project || {}).recruitUsers || []).indexOf(authUser.userId) !== -1;
 
-  const isStaff = authUser.isStaff === 1;
-
   const projectCardStatus = project && project.agileCardStatus;
 
   const showAddReviewButton =
-    (isReviewer || isStaff) &&
+    (isReviewer) &&
```

Comment:
We have a utility function that does this. Should we not call that function? I think it's called `getShowAddReviewButton`